src/roles/upgrader.js

```diff
@@ -1,13 +1,15 @@
+'use strict'
+
 const MetaRole = require('roles_meta')
 
-const CONTROLLER_MESSAGE = "* Self Managed Code * quorum.tedivm.com * #quorum in Slack *"
+const CONTROLLER_MESSAGE = '* Self Managed Code * quorum.tedivm.com * #quorum in Slack * (Lur was here)'
```

Comment:
nice try

Reply:
Yup, this is why i'm downvoting. Come on @lurr, cut the trolling 😛

Reply:
I'll also upvote once this is corrected 😉